libs/localization/src/lib/en.json

```diff
@@ -1773,6 +1775,7 @@
       "probabilityForClassSelectionHeader": "Probability for class",
       "targetSelectionHeader": "Target",
       "metricSelectionDropdownPlaceholder": "Select metrics to compare your cohorts.",
+      "classSelectionDropdownPlaceholder": "None selected",
```

Comment:
Can we provide string with an action, something like "Select classnames to use for a classname-based analysis"